posthog/session_recordings/session_recording_api.py

```diff
@@ -823,6 +851,76 @@ def _send_realtime_snapshots_to_client(
         else:
             raise exceptions.ValidationError(f"Invalid version: {version}")
 
+    @extend_schema(description="Generate session recording filters using AI")
+    @action(methods=["POST"], detail=False, url_path="ai/filters")
+    def ai_filters(self, request: Request, *args: Any, **kwargs: Any) -> Response:
+        if not request.user.is_authenticated:
```

Comment:
this might be unnecessary, we can write a test to prove if it's handled elsewhere

Reply:
shoujld definitely have some tests here!

i'm much more strict on tests in the backend than the front :)